schema.graphql

```diff
@@ -5,10 +5,15 @@ type Domain @entity {
   name: String
   labelHash: String
   parent: Domain
-  subdomains: [Domain!]
+  subdomains: [Domain!] @derivedFrom(field: "parent")
   owner: Account
   creator: Account
   transactionId: Bytes
+  lockedBy: Account
```

Comment:
May be null, please make `Account!`

Reply:
doing this causes a ✖ HTTP error deploying the subgraph 502
i suspect for a similiar reason that making metaData String! does even though its a different error

Reply:
Please disregard, i was confused by what `!` does